tidy3d/web/simulation_task.py

```diff
@@ -537,3 +537,11 @@ def get_log(
             verbose=verbose,
             progress_callback=progress_callback,
         )
+
+    def abort(self):
+        """Abort current task from server."""
+        if not self.task_id:
+            raise ValueError("Task id not found.")
+        return http.put("tidy3d/tasks/abort", json={"taskType": "NORMAL", "taskId": self.task_id})
```

Comment:
Can we use "FDTD" instead of "NORMAL"? This is not part of this PR as it affects multiple codebases, but could you make a JIRA ticket if needed? In the future we will have other task types too so let's be more clear about the naming.

Reply:
"taskType": "NORMAL/FDTD" , taskType is just an input parameter identification bit, and will not affect the code logic of the backend, ’NORMAL‘ can be freely modified to ’FDTD‘

Reply:
changed NORMAL to FDTD